tests/Cms/Media/MediaTest.php

```diff
@@ -106,13 +126,144 @@ public function testPublish()
         $this->assertFalse(is_dir($versionB1));
     }
 
+    /**
+     * @covers ::root
+     */
+    public function testRoot()
+    {
+        // copy test image to content
+        Dir::make($this->tmp . '/content');
+        F::copy($this->fixtures . '/../files/test.jpg', $this->tmp . '/content/test.jpg');
+
+        $kirby = $this->app;
+
+        // string model
+        $hash = '12345678';
+        $this->assertStringEndsWith('tests/Cms/Media/tmp/media/assets/assets/' . $hash, Media::root($kirby, 'assets', $hash));
```

Comment:
To ensure that `assets` doesn't end up there by accident:

```suggestion
        $this->assertStringEndsWith('tests/Cms/Media/tmp/media/assets/files' . $hash, Media::root($kirby, 'files', $hash));
```